keycloak/themes/hmda/login/login.ftl

```diff
@@ -15,7 +15,7 @@
                     <#if usernameEditDisabled??>
                         <input id="username" name="username" type="text" autocapitalize="off" autocorrect="off" value="${(login.username!'')?html}" disabled>
                     <#else>
-                        <input id="username" name="username" value="${(login.username!'')?html}" type="text" autofocus autocomplete="off" />
+                        <input id="username" name="username" value="${(login.username!'')?html}" type="text" autofocus />
```

Comment:
Yeah, just seems strange to me on the `username` field ... not totally sure what the security benefits are there, but for UX its very nice to have it on.